core/src/backends/eclair/rest/types.rs

```diff
@@ -1,10 +1,13 @@
 #![allow(clippy::from_over_into)]
 
+use crate::{
+    types::*,
+    utils::{self, msat_to_sat},
+};
```

Comment:
It might be better to import the `utils` crate, then call functions using `utils::function()`. That way we can clearly see we are calling an util function. Thoughts?
```suggestion
use crate::{
    types::*,
    utils,
};
```